gw_spaceheat/actors/strategy_switcher.py

```diff
@@ -21,9 +21,9 @@
 def strategy_from_node(node: ShNode):
     if not node.has_actor:
         return None
-    if node.role not in list(switcher.keys()):
-        raise Exception(f"Missing implementation for {node.role.value}!")
-    func = switcher[node.role]
+    if node.actor_class.value not in [key.value for key in switcher]:
+        raise Exception(f"Missing implementation for class {node.actor_class.value}, node {node.alias}!")
```

Comment:
This was the main bug I introduced when I introduced `ActorClass` (which is exactly supposed to give the name of the actor function) in order to decouple it from the concept of `Role`. I forgot to do this critical change.